cli/internal/cmdutil/cmdutil.go

```diff
@@ -155,15 +155,7 @@ func (h *Helper) GetCmdBase(cliConfig *turbostate.ParsedArgsFromRust) (*CmdBase,
 	if err != nil {
 		return nil, err
 	}
-	repoConfig, err := config.ReadRepoConfigFile(config.GetRepoConfigPath(repoRoot), cliConfig)
-	if err != nil {
-		return nil, err
-	}
-	userConfig, err := config.ReadUserConfigFile(h.UserConfigPath, cliConfig)
-	if err != nil {
-		return nil, err
-	}
-	remoteConfig := repoConfig.GetRemoteConfig(userConfig.Token())
```

Comment:
`userConfig.Token()` is used for more than just the remoteCache, so collapsing it down to just `remoteConfig` mayyy be confusing. Luckily, we will probably need a different token for remote cache and for other API endpoints soon, so this works in favor of that.

Reply:
Yes, but all of those uses as far as I can tell are now ported to Rust

Reply:
Moving to rust is fine, collapsing it to a remote config token is the issue. Eg the token is used to post runs to spaces

Reply:
yeah I like it.